js/users.js

```diff
@@ -722,6 +722,19 @@ $(document).ready(function () {
 	// TODO: move other init calls inside of initialize
 	UserList.initialize($('#userlist'));
 
+	OC.AppConfig.getValue('core', 'umgmt_set_password', 'false', function (data) {
+		var showPassword = $.parseJSON(data);
+		if (showPassword === true) {
+			$("#newuserpassword").show();
+			$("#newemail").hide();
+			$('#CheckBoxPasswordOnUserCreate').attr('checked', true);
```

Comment:
why is `Checkbox` capitalized and not the other ones ?

Reply:
The reason for using `CheckBox` capitalized is because, there are multiple places where `CheckBox` is used across the file. So tried to follow the same here.